prompt_cli.py

```
@@ -0,0 +1,462 @@
+        except subprocess.CalledProcessError:
+            print(f"❌ Editor '{editor}' failed")
+            return ""
+        except FileNotFoundError:
+            print(f"❌ Editor '{editor}' not found")
+            print("💡 Set EDITOR environment variable or use 'nano', 'vim', etc.")
+            return ""
+        finally:
+            try:
+                os.unlink(temp_path)
+            except:
+                pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)